files/2020/08/buck-stops-here.md

```diff
@@ -0,0 +1,35 @@
+# Governance: The Buck Stops Here
+
+This project uses the [buck stops here][buck-stops-here] rules for making
+decisions, also known as [benevolent dictator for life][bdfl].  The first
+section below describes these rules; the second lists decisions made to date.
+
+[buck-stops-here]: https://en.wikipedia.org/wiki/Buck_passing
+[bdfl]: https://en.wikipedia.org/wiki/Benevolent_dictator_for_life
+
+## Making Decisions
+
+1.  ____ is the project lead (PL) and has final say on all decisions in this
+    project.  They may consult when and as they think best, but any votes they
+    call are purely advisory.
+
+2.  The PL may delegate authority in whole or in part to anyone at any time, and
+    revoke any such delegation at their discretion.
+
+3.  To request a decision from the PL, file an issue in this repository tagged
+    "discussion".
+
+4.  All decisions, including delegation or revocation of authority, will be
+    recorded in this file.
+
+5.  The PL can change these rules at any time, and will announce changes in the
+    project's blog and by updating this file.
+
+## Recording Decisions
+
+Decisions that have been made are listed below in reverse chronological order.
+Each entry contains:
+
+-   the date the decision was made,
+-   the decisions's one-line summary, and
+-   a link to the decision's issue in GitHub.
```

Comment:
I loved this! This is a piece of information that can be REALLY useful if made easily reachable